junqo_back/src/users/mapper/user-mapper.ts

```
@@ -0,0 +1,19 @@
+      userModel.id,
+      userModel.type as UserType,
+      userModel.name,
+      userModel.email,
+      userModel.password,
+    );
+  }
+
+  public static toDomainUsers(userModels: UserModel[]): DomainUser[] {
+    return userModels.map((userModel) => this.toDomainUser(userModel));
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)